safe-global/safe-react-apps
# feat(wallet-connect): support `safe_setSettings`

## What it solves

The support of `safe_setSettings` in the WalletConnect app.

## How this PR fixes it

The dependencies have been updated, permitting the `safe_setSetting` RPC call.

## How to test it

Prerequisites:
- Open the https://github.com/safe-global/web-core/pull/1162 deployment.
- Run the [EIP-1271 test DApp](https://github.com/5afe/eip-1271-dapp) locally.

Method:
1. Open the WC app in the aforementioned deployment.
2. Connect to the aforementioned test DApp.
3. Toggle off-chain signing and attempt to sign a message.
4. Observe the off-chain signing modal.
5. Sign the message.
6. Observe the signed message in the "Signatures" tab of the transaction list.
